### PR TITLE
[FINAL][HOLD] iOS 11: Appstore Promoted IAP's

### PR DESCRIPTION
**Pull request by msun**
For: https://trello.com/c/CkY6oBbj/1721-ios11-in-app-purchases-from-the-app-store

Cherry-picked from 

PR: https://github.com/mattt/CargoBay/pull/88/
~~Commit: https://github.com/dcilia/CargoBay/commit/1062e55990f45282214993d382e1162369f1ef3f~~
Commit: https://github.com/dcilia/CargoBay/commit/35610367493255f7ce551aa22f690ccb81d29c55

### CODE REVIEW

File: CargoBay/CargoBay.h [hunk 1]
```diff
@@ -169,6 +169,16 @@
  */
 - (void)setPaymentQueueUpdatedDownloadsBlock:(void (^)(SKPaymentQueue *queue, NSArray *downloads))block;
 
+
+/**
+ Sets a block to be called when the payment has requested directly via the App Store.
```
Comment (antonmorant) [inline CargoBay/CargoBay.h:174]: the payment _was_ requested? Or _has been_ requested, maybe?

Comment (msun) [reply CargoBay/CargoBay.h:174]: I didn't write this code.
You can suggest further suggestions here: https://github.com/mattt/CargoBay/pull/88

Comment (antonmorant) [reply CargoBay/CargoBay.h:174]: done, thx

Comment (antonmorant) [reply CargoBay/CargoBay.h:174]: It's also not a big deal, btw.

**Pull request merged**